packages/analytics/src/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -42,11 +42,21 @@ Object {
   "FromParameterType": Object {
     "Bag": "Bag",
     "Editorial": "Editorial",
+    "Hero": "Hero",
+    "Homepage": "Homepage",
+    "MiniBag": "Mini Bag",
+    "MiniPdp": "Mini Pdp",
+    "MiniWishlist": "Mini Wishlist",
+    "Navigation": "Navigation",
```

Comment:
Question: What this from value is supposed to represent?

Reply:
The from type, should represent common locations from where analytics events are triggered.
Somehow in the list, could Navigation fit some requirement.

I can imagine for example some Navigation Widget, with some share page icon(will trigger share event), that could fit in this example.